mosec/protocol.py

```diff
@@ -117,4 +118,4 @@ def _recv_all(conn, length):
         packet = conn.recv_into(mv)
         mv = mv[packet:]
         size += packet
-    return buffer
+    return bytes(buffer)
```

Comment:
Same question.

Reply:
Otherwise, `bytearray` cannot be passed into `plasma.ObjectID()`, though it can be passed into `pickle.loads()`.